packages/blueprints-integration/src/userEditing.ts

```diff
@@ -43,3 +49,22 @@ export enum UserEditingType {
 	/** Form of selections */
 	FORM = 'form',
 }
+
+/**
+ * Grouping of schemas
+ */
+export interface UserEditingGroupingType {
```

Comment:
I wonder if these types should be closely coupled to the SourceLayers instead of being free form content from Blueprints.

Then it could just be a field on the Schema part of the CoreUserEditingDefinitionForm, i.e. `ui:sourceLayerID`. That would also support the double colour for voice overs.